REQUIRE

```diff
@@ -1,7 +1 @@
-julia 0.6
-AWSCore 0.3
```

Comment:
Direct dependencies still need to be listed in REQUIRE; it's used for constructing the registry.

Reply:
ah ok, updated